test/over_react/experimental/redux_component_test.dart

```diff
@@ -95,6 +96,31 @@ void main() {
       expect(component.numberOfRedraws, 1);
     });
 
+    test('properly redraws when isPure is true', () async {
+      var store = new Store<BaseState, BaseStateBuilder, BaseActions>(
+        baseReducerBuilder.build(),
+        baseState,
+        baseActions,
+      );
+      var jacket = mount<TestDefaultComponent>((TestDefault()..store = store)());
+      TestDefaultComponent component = jacket.getDartInstance();
+
+      store.actions.trigger1();
+      await new Future.delayed(Duration.ZERO);
+      expect(component.numberOfRedraws, 1);
+
+      jacket.rerender((TestDefault()
+        ..store = store
+        ..id = 'new id'
```

Comment:
Is the purpose of this ID to change some unrelated props when rerendering? If so, it might be nice to call that out explicitly, perhaps in the reason of the next `expect`.

```dart
expect(component.numberOfRedraws, 1, 
    reason: 'should not redraw when rerendered, even if other props change');
```